linkerd/app/outbound/src/lib.rs

```diff
@@ -518,17 +518,23 @@ impl Config {
             .push_map_target(|a: listen::Addrs| a.target_addr())
             .into_inner();
 
-        let http = http::DetectHttp::new(
+        const DETECT_BUFFER_CAPACITY: usize = 8192;
```

Comment:
nit: if this has the same value between inbound and outbound, and is unlikely to change or be configurable, why not just stick the constant in `transport` and have the prefix layer always use that value?